app/frontend/tokenRegistry/tokenRegistry.ts

```diff
@@ -22,6 +24,12 @@ export class TokenRegistry {
   private readonly _fetchTokensMetadata = async (
     subjects: string[]
   ): Promise<TokenMetadataResponse> => {
+    if (subjects.length > MAX_REQUEST_SIZE) {
```

Comment:
MAX_REQUEST_SIZE can be mistook for the byte size, something like MAX_SUBJECTS_COUNT would be less ambiguous

Reply:
Refactored